roles/debian/ssl/tasks/letsencrypt.yml

```diff
@@ -35,6 +35,26 @@
   when:
     - ssl.letsencrypt.venv_install_username is defined
 
+- name: Set LE email if defined globally.
+  ansible.builtin.set_fact:
+    _le_email: "{{ nginx.le_email }}"
+  when: nginx.le_email is defined
+
+- name: Set LE email if defined per item.
+  ansible.builtin.set_fact:
+    _le_email: "{{ ssl.email }}"
+  when: ssl.email is defined
+
+- name: Set LE cron if defined globally.
+  ansible.builtin.set_fact:
+    _le_cron: "{{ nginx.le_cron }}"
```

Comment:
Undocumented variable. Should be commented out in `roles/debian/nginx/defaults/main.yml`.